style.css

```diff
@@ -0,0 +1,117 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&family=Roboto:wght@400;500;700;900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  list-style: none;
+}
+
+:root {
+  --white-color: #fff;
+  --text-colored-blue: #6070ff;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  background-color: var(--white-color);
+  font-family: Poppins;
+  overflow-x: hidden;
+}
+
+header {
+  width: 100%;
+  position: fixed;
+  top: 0;
+  padding: 2rem;
+}
+
+header h1 {
+  font-size: 1.125rem;
+  font-weight: 700;
+}
+
+.container {
+  width: 90%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: 1.25rem;
+  text-align: center;
+  color: var(--text-colored-blue);
+}
+
+#about,
+#contact {
+  height: 700px;
+  width: 100%;
+}
+
+.portfolio-header {
+  display: flex;
+  font-size: 3rem;
+  text-align: center;
+  font-weight: 700;
+}
+
+#portfolio {
+  margin: 4rem 0 0 0;
+  width: 100%;
+  background: url('images/portfolio.png') no-repeat;
+}
```

Comment:
- [ ] Good job including a background on your web pages 😄 , however make sure the background you use look the same as in the Figma 👍 

|  Your Design |  Figma Design |
| :---: | :---: |
| ![Capture 3](https://user-images.githubusercontent.com/80458199/228262858-81490d37-2b68-4ffd-8065-4cc3e09a711f.PNG) |  ![Capture](https://user-images.githubusercontent.com/80458199/228262916-3dc786f8-e8ad-4cf6-b970-11b9d4b2e75b.PNG) |